includes/admin/views/tmpl-order-actions.php

```diff
@@ -13,6 +13,12 @@
 $is_override      = edd_is_order_refundable_by_override( $order->id );
 $is_window_passed = edd_is_order_refund_window_passed( $order->id );
 
+// EDD does not support initializing refunds through the Amazon gateway.
```

Comment:
@cklosowski I'm reconsidering having this set here--it's not necessary to prevent the call to Amazon, but does prevent a store owner from issuing a partial refund at the gateway and updating the order here accordingly. What if we allowed the refund form to still initialize, but use the notice area to show that it's a two part process?